manager/controlapi/node.go

```diff
@@ -273,7 +273,8 @@ func (s *Server) RemoveNode(ctx context.Context, request *api.RemoveNodeRequest)
 		if node == nil {
 			return grpc.Errorf(codes.NotFound, "node %s not found", request.NodeID)
 		}
-		if node.Spec.Role == api.NodeRoleManager {
+		member := s.raft.GetMemberByNodeID(request.NodeID)
```

Comment:
any chances that `s.raft` could be `nil`? I find the check of `if s.raft != nil` in the beginning of the function